source/common/stats/utility.cc

```diff
@@ -16,9 +17,16 @@ std::string Utility::sanitizeStatsName(absl::string_view name) {
   if (absl::StartsWith(name, ".")) {
     name.remove_prefix(1);
   }
+
   std::string stats_name = std::string(name);
   std::replace(stats_name.begin(), stats_name.end(), ':', '_');
   std::replace(stats_name.begin(), stats_name.end(), '\0', '_');
+  absl::StrReplaceAll(
+      {
+          {"_//", "_"},
+          {"_/", "_"},
```

Comment:
I think you could make this work with a single StrReplaceAll -- just use this order:
```
   ://   -->  _
   :/    -->  _
   :     -->  _
   \0    -->  _
```
that way you are not depending on chaining.

I still need to see whether this has a measurable impact on performance in the thread_local_store_speed_test. Run that on an otherwise unloaded machine, with `-c opt`, with and without this change.

Reply:
Not really, StrReplaceAll looks do not support replacing one match multiple times :(

Reply:
How does this case differ from https://github.com/envoyproxy/envoy/blob/75aa6b5e6ff8ee5cb50b3b613dfb0d7514b60eb6/source/common/html/utility.cc#L12 ?

The problem with your earlier form is you were saying "convert a->b, b->c", and expecting "a" to become "c". With my suggestion you're just converting directly a->c and that's fine.

Reply:
I misunderstood it.